python/probe_screen.py

```diff
@@ -380,18 +393,19 @@ def on_btn_jog_pressed( self, widget, data = None ):
         else:
             value = 1
 
-        velocity = float(self.inifile.find("TRAJ", "DEFAULT_VELOCITY"))
+        velocity = float(self.inifile.find("TRAJ", "DEFAULT_LINEAR_VELOCITY"))
 
         dir = widget.get_label()[1]
         if dir == "+":
             direction = 1
         else:
             direction = -1
 
+        self.command.teleop_enable(1)
         if self.distance <> 0:  # incremental jogging
-            self.command.jog( linuxcnc.JOG_INCREMENT, axisnumber, direction * velocity, self.distance )
+            self.command.jog( linuxcnc.JOG_INCREMENT, False, axisnumber, direction * velocity, self.distance )
         else:  # continuous jogging
-            self.command.jog( linuxcnc.JOG_CONTINUOUS, axisnumber, direction * velocity )
+            self.command.jog( linuxcnc.JOG_CONTINUOUS, False, axisnumber, direction * velocity )
```

Comment:
The other 2.8 PR open has self`.command.teleop_enable(0)` and `True` where I've used false. With a dual motor / gantry axis, using teleop=0/False results in putting linuxCNC back into joint mode which can cause only a single motor of the pair to run.